app/controllers/users.controller.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
```

Comment:
Why are you using `strict mode`?

Reply:
I have removed strict mode.

Reply:
Ok. I was not saying it is wrong, I just want to check if you know why you were using it 😛

Reply:
I know that strict mode eliminates some behaviors that normal mode allows, i think it is good idea to get errors that could be written by mistake.😅

Reply:
Good enough 😄